application-admintools-ui/src/main/resources/AdminTools/Code/HealthCheckResources.xml

```diff
@@ -51,13 +53,16 @@
 
 #macro (getHealthCheckResources)
   #set ($jobMinJsURL = $xwiki.getSkinFile('uicomponents/job/job.min.js'))
-  #set ($healthCheckJobRequest = $services.admintools.createJobRequest($services.wiki.getCurrentWikiId()))
-  #set ($healthJobStatus = $services.job.getJobStatus($healthCheckJobRequest.id))
+  #set ($jobId = ['adminTools', 'healthcheck', $services.wiki.getCurrentWikiId()])
+  #set ($healthJobStatus = $services.job.getJobStatus($jobId))
   {{html clean=false}}
-   &lt;div hidden>
-     &lt;input type="text" id="jobMinJsURL" name="jobMinJsURL" value=$jobMinJsURL>&lt;/input>
+   &lt;div hidden&gt;
+   #data($jobMinJsURL)
+     &lt;jobJsURL
```

Comment:
You have a DIV wrapper. Why not mark it with a know CSS class and put the metadata there, instead of inventing custom HTML elements?